### PR TITLE
feat(PDEV-1827): Added v10 POST route for image uploading during onboarding

### PR DESCRIPTION
**Pull request by prachisshah**

### CODE REVIEW

**Comment by prachisshah** [comment]
Platform code: https://github.com/trizic/platform/pull/1921

File: api/v10/index.raml [hunk 1]
```diff
@@ -325,6 +325,37 @@
 
                 Invalid firmGuid, invalid custodian, invalid account type
 
+      /image:
+        post:
+          description: |
+            ### Submit an image in support of the FINRA letter requirement during account onboarding. Supported on Apex.
+          queryParameters:
+            custodian:
```
Comment (jamesdaniello) [inline api/v10/index.raml]: these queryParameters don't seem to match the PR in platform https://github.com/trizic/platform/pull/1921

Comment (prachisshah) [reply api/v10/index.raml]: Updated trizic/platform#1921 to add request parameter fileType.

Comment (prachisshah) [reply api/v10/index.raml]: Updated trizic/platform#1921 to add request parameter fileType.

Comment (jamesdaniello) [inline api/v10/index.raml]: custodian comes in as a uriParameter up on line 183

Comment (prachisshah) [reply api/v10/index.raml]: Removed.

**Comment by prachisshah** [comment]
Merging, closing PR, deleting branch.

**Pull request merged**